templates/cisco_ios_show_ip_eigrp_neighbors.template

```diff
@@ -0,0 +1,11 @@
+Value ADDRESS (\d+\.\d+\.\d+\.\d+)
+Value INTERFACE (\w+(\d+|\d+/\d+|\d+/\d+/\d+))
+Value HOLD (\d+)
+Value UPTIME ((\d+:\d+:\d+|\d+\w\d+\w))
+Value SRTT (\d+)
+Value RTO (\d+)
+Value Q_CNT (\d+)
+Value SEQ_NUM (\d+)
+
+Start
+  ^.*\s+${ADDRESS}\s+${INTERFACE}\s+${HOLD}\s+${UPTIME}\s+${SRTT}\s+${RTO}\s+${Q_CNT}\s+${SEQ_NUM} -> Record
```

Comment:
```suggestion
 ^.*\s+${ADDRESS}\s+${INTERFACE}\s+${HOLD}\s+${UPTIME}\s+${SRTT}\s+${RTO}\s+${Q_CNT}\s+${SEQ_NUM}\s*$$ -> Record
  ^\s*(?:IP-|)EIGRP(?:-IPv4|)\s+[Nn]eighbors\s+for\s+(?:process|AS\()\s+${AS}(?:\)|)\s*$$
  ^\s*$$
  ^\s*H\s+Address\s+Interface\s+Hold\s+Uptime\s+SRTT\s+RTO\s+Q\s+Seq\s*$$
  ^\s+\(sec\)\s+\(ms\)\s+Cnt\s+Num\s*$$
  ^\s*Load\s+for
  ^Time\s+source\s+is
  ^. -> Error
  
```

Reply:
The deleted space should not be deleted, but GitHub would not let me do two spaces there; it was either one space or a newline